.circleci/config.yml

```diff
@@ -1,5 +1,7 @@
-# generator: n-circle2-cli
-# template: component
+version: 2.1
+
+orbs:
+  ft-snyk-orb: financial-times/ft-snyk-orb@0
```

Comment:
Might wanna use [Change API orb](https://github.com/Financial-Times/change-api-orb) too to notify on usual team channels upon releases

Reply:
Sure, is this something other components do on release?

Reply:
ah right, this is a package. We do not currently support logging release for packages via Change API, so maybe not 😅